notebooks/214-grammar-correction/214-grammar-correction-quantize.ipynb

```diff
@@ -0,0 +1,1147 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip install -q "git+https://github.com/openvinotoolkit/nncf.git@9c671f0ae0a118e4bc2de8b09e66425931c0bfa4"
</pre><p>I had to use NNCF develop, because on NNCF 2.6 there are random accuracy drops after quantization.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>